ruby/alias_manager.rb

```diff
@@ -0,0 +1,67 @@
+# write a method that takes a spy's real name (e.g., "Felicia Torres")
+# and creates a fake name with it by doing the following:
+# Swapping the first and last name
+
+# Change all of the vowels to the next vowel in 'aeiou'
+def change_vowels(vowel)
+  vowels = 'aeioua'
```

Comment:
'a' the beginning and end is clever but error prone.

just use conditionals for the edge case